From ff8de9f1c7d6060281a2bf46eebeb1f1568d57e2 Mon Sep 17 00:00:00 2001
From: Neil Cheng <Neil.Cheng@quantatw.com>
Date: Mon, 17 Aug 2026 14:06:04 +0800
Subject: [PATCH] configurations: rainier: Update FSC v2 configs

Adjust thermal PID and stepwise fan control parameters for both
rainier_scm_2600 and rainier_scm_2700 configurations:

- Modified the stepwise settings for SCM_INLET_TEMP_C and MB_CPU_TEMP_C
- Updated the MB_CPU_TEMP_C setpoint to 85
- PID_NIC input sensor name to regex NIC_(TEMP|Temp)_4
- Remove Stepwise_OTHERS entry (MB_INLET/OUTLET, PDB_L/R_TEMP_C)

Upstream-Status: Submitted [https://gerrit.openbmc.org/c/openbmc/entity-manager/+/93575]

Change-Id: I2172bbeba8640ae20064f1a6252d31ec3e2fc9e7
Signed-off-by: Neil Cheng <Neil.Cheng@quantatw.com>
---
 .../meta/rainier/rainier_scm_2600.json        | 42 ++++---------------
 .../meta/rainier/rainier_scm_2700.json        | 42 ++++---------------
 2 files changed, 16 insertions(+), 68 deletions(-)

diff --git a/configurations/meta/rainier/rainier_scm_2600.json b/configurations/meta/rainier/rainier_scm_2600.json
index 06eb13b..fa8e6af 100644
--- a/configurations/meta/rainier/rainier_scm_2600.json
+++ b/configurations/meta/rainier/rainier_scm_2600.json
@@ -250,7 +250,7 @@
             "OutLimitMin": 0,
             "PCoefficient": -5.0,
             "PositiveHysteresis": 0.0,
-            "SetPoint": 76.0,
+            "SetPoint": 85.0,
             "SlewNeg": 0.0,
             "SlewPos": 0.0,
             "Type": "Pid",
@@ -395,7 +395,7 @@
             "ILimitMax": 100,
             "ILimitMin": -50,
             "Inputs": [
-                "NIC_Temp_4"
+                "NIC_(TEMP|Temp)_4"
             ],
             "Name": "PID_NIC",
             "NegativeHysteresis": 2.0,
@@ -430,9 +430,9 @@
             "NegativeHysteresis": 2.0,
             "Output": [
                 22.0,
-                28.0,
+                26.0,
+                30.0,
                 34.0,
-                40.0,
                 95.0
             ],
             "PositiveHysteresis": 0.0,
@@ -457,11 +457,9 @@
             "NegativeHysteresis": 3.0,
             "Output": [
                 22.0,
+                26.0,
+                30.0,
                 34.0,
-                38.0,
-                42.0,
-                46.0,
-                50.0,
                 95.0
             ],
             "PositiveHysteresis": 0.0,
@@ -470,9 +468,7 @@
                 67.0,
                 69.0,
                 71.0,
-                73.0,
-                75.0,
-                82.0
+                91.0
             ],
             "Type": "Stepwise",
             "Zones": [
@@ -601,7 +597,7 @@
         {
             "Class": "temp",
             "Inputs": [
-                "NIC_Temp_4"
+                "NIC_(TEMP|Temp)_4"
             ],
             "Name": "Stepwise_NIC",
             "NegativeHysteresis": 2.0,
@@ -621,28 +617,6 @@
                 "Zone 1"
             ]
         },
-        {
-            "Class": "temp",
-            "Inputs": [
-                "MB_INLET_TEMP_C",
-                "MB_OUTLET_TEMP_C",
-                "PDB_L_TEMP_C",
-                "PDB_R_TEMP_C"
-            ],
-            "Name": "Stepwise_OTHERS",
-            "NegativeHysteresis": 0.0,
-            "Output": [
-                10.0
-            ],
-            "PositiveHysteresis": 0.0,
-            "Reading": [
-                5.0
-            ],
-            "Type": "Stepwise",
-            "Zones": [
-                "Zone 1"
-            ]
-        },
         {
             "Address": "0x4B",
             "Bus": 9,
diff --git a/configurations/meta/rainier/rainier_scm_2700.json b/configurations/meta/rainier/rainier_scm_2700.json
index 6b8e879..415698f 100644
--- a/configurations/meta/rainier/rainier_scm_2700.json
+++ b/configurations/meta/rainier/rainier_scm_2700.json
@@ -316,7 +316,7 @@
             "OutLimitMin": 0,
             "PCoefficient": -5.0,
             "PositiveHysteresis": 0.0,
-            "SetPoint": 76.0,
+            "SetPoint": 85.0,
             "SlewNeg": 0.0,
             "SlewPos": 0.0,
             "Type": "Pid",
@@ -461,7 +461,7 @@
             "ILimitMax": 100,
             "ILimitMin": -50,
             "Inputs": [
-                "NIC_Temp_4"
+                "NIC_(TEMP|Temp)_4"
             ],
             "Name": "PID_NIC",
             "NegativeHysteresis": 2.0,
@@ -496,9 +496,9 @@
             "NegativeHysteresis": 2.0,
             "Output": [
                 22.0,
-                28.0,
+                26.0,
+                30.0,
                 34.0,
-                40.0,
                 95.0
             ],
             "PositiveHysteresis": 0.0,
@@ -523,11 +523,9 @@
             "NegativeHysteresis": 3.0,
             "Output": [
                 22.0,
+                26.0,
+                30.0,
                 34.0,
-                38.0,
-                42.0,
-                46.0,
-                50.0,
                 95.0
             ],
             "PositiveHysteresis": 0.0,
@@ -536,9 +534,7 @@
                 67.0,
                 69.0,
                 71.0,
-                73.0,
-                75.0,
-                82.0
+                91.0
             ],
             "Type": "Stepwise",
             "Zones": [
@@ -667,7 +663,7 @@
         {
             "Class": "temp",
             "Inputs": [
-                "NIC_Temp_4"
+                "NIC_(TEMP|Temp)_4"
             ],
             "Name": "Stepwise_NIC",
             "NegativeHysteresis": 2.0,
@@ -687,28 +683,6 @@
                 "Zone 1"
             ]
         },
-        {
-            "Class": "temp",
-            "Inputs": [
-                "MB_INLET_TEMP_C",
-                "MB_OUTLET_TEMP_C",
-                "PDB_L_TEMP_C",
-                "PDB_R_TEMP_C"
-            ],
-            "Name": "Stepwise_OTHERS",
-            "NegativeHysteresis": 0.0,
-            "Output": [
-                10.0
-            ],
-            "PositiveHysteresis": 0.0,
-            "Reading": [
-                5.0
-            ],
-            "Type": "Stepwise",
-            "Zones": [
-                "Zone 1"
-            ]
-        },
         {
             "Address": "0x4B",
             "Bus": 9,
